.github/pull_request_template.md

```diff
@@ -5,11 +5,34 @@
 ## How
 *Describe the solution*
 
+## Recommended reading order
+1. `x.java`
+2. `y.python`
+
 ## Pre-merge Checklist
-- [ ] *Run integration tests*
-- [ ] *Publish Docker images*
+Expand the checklist which is relevant for this PR. 
+
+<details><summary> <strong> Connector checklist </strong> </summary>
+<p>
+
+- [ ] Issue acceptance criteria met
+- [ ] Unit & integration tests added as appropriate (and are passing)
+    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
+- [ ] `/test` command documented [here]() is passing. 
```

Comment:
Why don't we write the whole command in the checklist (with the long form so it is compatible with publish)?
`/test connector=connectors/<name>`

So we can just copy/paste/replace the name and not remember the correct syntax